falak/pkg/prayer/client/client.go

```diff
@@ -0,0 +1,50 @@
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+
+	"github.com/jadwalapp/symmetrical-spoon/falak/pkg/httpclient"
+)
+
+type client struct {
+	cli httpclient.HTTPClient
+}
+
+func (a *client) GetPrayertimeInfo(ctx context.Context, r *GetPrayertimeInfoRequest) (*GetPrayertimeInfoResponse, error) {
+	encodedCity := url.QueryEscape(r.City)
+	encodedCountry := url.QueryEscape(r.Country)
+	url := fmt.Sprintf("https://api.aladhan.com/v1/timingsByCity?city=%s&country=%s&date=%s", encodedCity, encodedCountry, r.Date)
```

Comment:
put this in the client struct as a `baseUrl`: `https://api.aladhan.com`

look at google client as an example.